torch/_inductor/codegen/triton.py

```diff
@@ -3385,6 +3419,7 @@ def store(
         if not self.inside_reduction and self.cooperative_reduction:
             exit_stack.enter_context(self.guard_cooperative_store(name, self.stores))
 
+        self._handle_pdl_before_access(self.stores, name, consider_reads=True)
```

Comment:
Right, so if the prior kernel reads from this buffer then we need to wait prior to write. Relatedly - when we do signal gdc_launch_dependents ? Wouldn't we also need to look at codegen schedule ?